src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
If you want to check the package, you can do it with `npm run lint`, if that says your code is okay, then it should be fine.